src/process_utility.c

```diff
@@ -5062,6 +5065,7 @@ timescaledb_ddl_command_start(PlannedStmt *pstmt, const char *query_string, bool
 	if (altering_timescaledb || !ts_extension_is_loaded_and_not_upgrading())
 	{
 		prev_ProcessUtility(&args);
+		ts_process_utility_context_reset();
```

Comment:
What about move it to inside `prev_processUtility` ?

Reply:
Yes, moved the context reset into `prev_processUtility`.